chef/automate
# automate infra proxy cookbook list UI

### :nut_and_bolt: Description: What code changed, and why?
Code changes for the introduction of infra proxy cookbook list component in infra proxy module to make automate-ui a single source of truth for chef infra cookbook data here

In this PR following points we have covered:

1. Created cookbook-list component in infra-proxy module to show cookbook list of specific Org
Designed infra proxy cookbook-list UI flows and used Infra-proxy-service
2. Fixed chef-sever left side navbar option issue for non accessible policy 


### :chains: Related Resources
Refered Wireframes: https://chef.invisionapp.com/share/3JVLVFMASDZ#/screens/401258847 
For feature description refer: #1544
### :+1: Definition of Done
1. Added cookbook-list component in infra proxy module 
2.  Fixed chef-sever left side navbar option issue for non accessible policy 
### :athletic_shoe: How to Build and Test the Change
build components/automate-ui
1. Unable chef server side nav option under Infrastructure using feature flag 
2. add sever and org credentials of chef-sever/chef manage
3. click on specific org to check cookbook list 
### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
![infra-cookbooklist](https://user-images.githubusercontent.com/54940695/75785381-a0543f80-5d89-11ea-8f4c-5554e3a5cfc1.png)
